package.json

```diff
@@ -28,6 +26,10 @@
   },
   "dependencies": {
     "@cypress/code-coverage": "^3.10.7",
+    "@fortawesome/fontawesome-svg-core": "^6.4.0",
+    "@fortawesome/free-regular-svg-icons": "^6.4.0",
```

Comment:
haven't used fortawesome before, but do you use `regular-svg-icons` ever? it appears you only ever get icons from `solid`.